akiraux/Akira
# Add fill panel for single and multiple items

## Summary / How this PR fixes the problem?


This PR will restore the FillPanel, for single and multiple items. 

In the beginning I though I restored also the BorderPanel, but in the end I think this PR is wide enough for it to be tested separately, and the smaller the change the better :)

## Steps to Test
 

* Create one or more fillable items (rect or ellipses, for example)
* Select one of them
* Change the color by editing the ColorRow
* Select multiple items, now the ColorRow should show a question mark on the color. Choose a color to change it for every selected item

## Screenshots 



Single item color change

https://user-images.githubusercontent.com/11556031/147827072-41ab1772-5c7a-438a-bf88-b20cd04fe0fc.mp4

Multiple items with same fill/color

https://user-images.githubusercontent.com/11556031/147827078-4e0c51db-d32e-4729-9b4f-6d6eeb25a94d.mp4

Multiple items with different colors

https://user-images.githubusercontent.com/11556031/147827081-a1597743-afbb-48bd-978e-f814ee61e8b5.mp4


## Known Issues / Things To Do




This update consider just the case of items with one fill. The add multiple fill option is not working, and neither the delete or hide fill. Only one fill at a time is working atm. 

I think that, for now, this should be enough, we can add it later, tell me what you think
